docs/editions.rst

```diff
@@ -28,40 +28,60 @@ limitations.
 
 .. _standard-edition:
 
-CrateDB
-=======
+CrateDB Standard Edition
+========================
+
+Take note of the following items when using the standard edition of CrateDB. 
 
 .. _enterprise-features:
 
 Enterprise features
 -------------------
 
-- :ref:`administration_user_management`: manage multiple database users
-- :ref:`administration-privileges`: configure user privileges
-- :ref:`admin_auth`: manage your database with authentication, and
-  more
-- System information functions: :ref:`CURRENT_USER <current_user>`,
-  :ref:`USER <user>`, :ref:`SESSION_USER <session_user>`
-- :ref:`Support for JavaScript in UDF <udf_lang_js>`: write user-defined
-  functions in JavaScript
-- :ref:`jmx_monitoring`: monitor your query stats with JMX
-- :ref:`aggregation-hll-distinct`: distinct count aggregation using the
-  `HyperLoglog++`_ algorithm
-- :ref:`window-function-firstvalue`: ``first_value`` window function
-- :ref:`window-function-lastvalue`: ``last_value`` window function
-- :ref:`window-function-lag`: ``lag`` window function
-- :ref:`window-function-lead`: ``lead`` window function
-- :ref:`window-function-nthvalue`: ``nth_value`` window function
-- `The CrateDB admin UI`_: `shards browser`_, `monitoring overview`_,
-  `privileges browser`_
+- :ref:`administration_user_management` for multiple database users
+- Configure user :ref:`privileges <administration-privileges>`
+- Manage your database with :ref:`authentication <admin_auth>` and more
+- Write :ref:`user-defined functions in JavaScript <udf_lang_js>`
+- Take advantage of these features in the `CrateDB admin UI`_:
+
+  - `shards browser`_
+
+  - `monitoring overview`_ 
+
+  - `privileges browser`_
+
+- Unlock the following :ref:`system information functions <scalar-sysinfo>`:
+
+  - :ref:`CURRENT_USER <current_user>`
+
+  - :ref:`USER <user>`
+
+  - :ref:`SESSION_USER <session_user>`
+
+- :ref:`Monitor <jmx_monitoring>` your query stats with JMX
+- Access to the :ref:`aggregation-hll-distinct` :ref:`aggregate function 
+  <aggregate-functions>`
+- Access to the following :ref:`window functions <window-functions>`:
+
+  - :ref:`window-function-firstvalue`
+
+  - :ref:`window-function-lastvalue`
+
+  - :ref:`window-function-lag`
+
+  - :ref:`window-function-lead`
+
+  - :ref:`window-function-nthvalue`
+
 
 .. _node-limitations:
 
 Node limitation
 ---------------
 
-To make full use of CrateDB, you must `acquire an Enterprise License`_. Unless and
-until you do that, CrateDB is limited to running on no more than three nodes.
+To make full use of CrateDB, you must `acquire an Enterprise License`_. The
+standard edition of CrateDB is limited to running on no more than three nodes
```

Comment:
"standard edition" usage here also depends on the outcome of my prior comment.

I